packages/ldap-auth/README.md

```diff
@@ -1 +1,96 @@
 # @immobiliarelabs/backstage-plugin-ldap-auth
+
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier?style=flat-square)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
+![license](https://img.shields.io/github/license/immobiliare/backstage-plugin-ldap-auth?style=flat-square)
+![npm (scoped)](https://img.shields.io/npm/v/@immobiliarelabs/backstage-plugin-ldap-auth?style=flat-square)
+
+> Customizable Authentication backend provider for LDAP servers for your [Backstage](https://backstage.io/) deployment
+
+Works either on simple stand-alone process or scaled infrastracture spanning multiple deployments using the shared PostgreSQL instance that Backstage already uses!
+
+This plugin is not meant to be used alone but with in pair with:
+
+-   The official [@backstage/plugin-catalog-backend-module-ldap](https://www.npmjs.com/package/@backstage/plugin-catalog-backend-module-ldap) which keeps in sync your LDAP users with Backstage user catalogs!
+-   Its sibling backend package [@immobiliarelabs/backstage-plugin-ldap-auth-backend](https://www.npmjs.com/package/@immobiliarelabs/backstage-plugin-ldap-auth-backend)
+
+Supports Node.js `>=14.0.0`
+
+## Table of Content
+
+<!-- toc -->
+
+-   [Installation](#installation)
+-   [Configuration](#configuration)
+-   [Powered Apps](#powered-apps)
+-   [Support & Contribute](#support--contribute)
+-   [License](#license)
+
+<!-- tocstop -->
+
+## Installation
+
+> These packages are available on npm.
+
+You can install them in your backstage installation using `yarn workspace`
+
+```bash
+# install yarn if you don't have it
+$ npm install -g yarn
+# install frontend plugin
+$ yarn workspace app add @immobiliarelabs/backstage-plugin-ldap-auth
+# install backend plugin
+$ yarn workspace backend add @immobiliarelabs/backstage-plugin-ldap-auth-backend
+```
+
+## Configuration
+
+> The react components accepts childrens to allow you to customize the login page look and feel
+
+The component out of the box only shows the form, but you can pass down children components to render your logos/top bar os whatever you want!
+
+<img src="./screen.jpg" width="600px" align="center"/>
+
+<br />
+
+In the `App.tsx` file change the `createApp` function to provide use our custom `SignInPage` in the `components` key.
```

Comment:
`to provide use` -> `to provide the use of `

Reply:
I've changed the whole sentence since it didn't make sense! thx